rpc/version.go

```diff
@@ -0,0 +1,9 @@
+package rpc
+
+import "context"
+
+func (provider *Provider) SpecVersion(ctx context.Context) (string, error) {
```

Comment:
Shouldn't this be added to `RpcProvider` interface as well?

Reply:
Could it also be added in the readme as a supported method?

Reply:
It would also be nice if the comments for this method were added (using the same format as in #415) so we don't have to it later